src/Common/tests/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -139,6 +137,7 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
             else
             {
+                //Desktop activation
```

Comment:
We do run tests in desktop with this, but I guess we can get rid of this now until we figure out a better alternative?